types/urijs/index.d.ts

```diff
@@ -210,11 +192,11 @@ declare namespace uri {
         encode(str: string): string;
         encodeQuery(qry: string): string;
         encodeReserved(str: string): string;
-        expand(template: string, vals: Object): URI;
+        expand(template: string, vals: object): URI;
 
         iso8859(): void;
 
-        joinPaths(...paths: (string | URI)[]): URI;
+        joinPaths(...paths: Array<string | URI>): URI;
```

Comment:
why in `hasSearch` prefer `Type[]` and here prefer `Array<Type>`?

Reply:
tslint error, complex types must use Array<...>